gateway/src/api/latest.rs

```diff
@@ -95,6 +95,7 @@ impl StatusResponse {
     }
 }
 
+#[instrument]
```

Comment:
In theory these should not be needed on endpoints, but maybe the tracing layer has issues :smile:

Reply:
Ah, I see. Shall we remove them from all the endpoints then? I will merge this now either way, though.